docs/docs-beta/docs/dagster-plus/deployment/management/managing-compute-logs-and-error-messages.md

```diff
@@ -3,4 +3,102 @@ title: Managing compute logs and error messages
 unlisted: true
 ---
 
-{/* TODO move from https://docs.dagster.io/dagster-plus/managing-deployments/controlling-logs */}
\ No newline at end of file
+import ThemedImage from '@theme/ThemedImage';
+
+:::note
+This guide is applicable to Dagster+.
+:::
+
+In this guide, we'll cover how to adjust where Dagster+ compute logs are stored and manage masking of error messages in the Dagster+ UI.
+
+By default, Dagster+ ingests [structured event logs and compute logs](/concepts/logging#log-types) from runs and surfaces error messages from [code locations](/guides/deploy/code-locations/) in the UI.
+
+Depending on your organization's needs, you may want to retain these logs in your own infrastructure or mask error message contents.
+
+---
+
+## Modifying compute log storage
+
+Dagster's compute logs are handled by the configured [`ComputeLogManager`](/api/python-api/internals#compute-log-manager). By default, Dagster+ utilizes the `CloudComputeLogManager` which stores logs in a Dagster+-managed Amazon S3 bucket, but you can customize this behavior to store logs in a destination of your choice.
+
+### Writing to your own S3 bucket
+
+If using the Kubernetes agent, you can instead forward logs to your own S3 bucket by using the [`S3ComputeLogManager`](/\_apidocs/libraries/dagster-aws#dagster_aws.s3.S3ComputeLogManager).
```

Comment:
Missing `S3ComputeLogManager` API docs.

It looks like we're missing _all_ integration docs, unless I'm mistaken.

Reply:
I think that one is here: https://docs-preview.dagster.io/api/python-api/libraries/dagster-aws#s3 -- the annoying thing is that none of the classes have anchors so can't be linked to :(

Reply:
Thank you.

We can likely improve the anchors, the mdx generated is being done by us in this custom sphinx extension:

https://github.com/dagster-io/dagster/tree/master/docs/sphinx/_ext/sphinx-mdx-builder

Pedram was the primary developer on it, with me assisting.

Reply:
Actually, it looks like the anchor links work (meaning https://docs-preview.dagster.io/api/python-api/libraries/dagster-aws#dagster_aws.s3.S3ComputeLogManager goes to the right place), it's just that those subclasses don't have the nice `#` icon on hover. We should just be able to replace `\_apidocs/` with `api/python-api/` and those links should work as expected, and I can make a ticket to update the custom sphinx extension to add anchor icons to subclasses.